src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -28,42 +28,28 @@
     static private Logger logger = java.util.logging.Logger.getLogger(PipelineSnippetGenerator.class.getClass().getName());
     private Launcher launcher;
 
-    PipelineSnippetGenerator(Launcher launcher){
-        this.launcher = launcher;
+    PipelineSnippetGenerator(){
+        this.launcher = Jenkins.get().createLauncher(null);
```

Comment:
this is the kind of change that should be in a different PR

the main reason for this, is as you start releasing, if there is a problem with either the logic of the tab removal, or changing the default launcher, it will be easier to distinguish which change caused the problem. because even though I don't think either of these changes together will break anything, when the plugin is used in the real world, there'll be use cases that you can't anticipate on how such a thing might break the functionality.

Reply:
I also suggest removing it. Passing `null` TaskListener is a bad practice which needs to be justified